threedigrid_builder/grid/cross_section_definitions.py

```diff
@@ -43,36 +49,47 @@ def convert(self, ids):
             content_pk=ids,
             code=self.code[idx],
             count=0,
+            count_yz = 0,
```

Comment:
Can you run the `black` code formatter? There's a `pre-commit` configuration in this repo, so after you installed the pre-commit tool (https://pre-commit.com/), you have to do `pre-commit install` inside the repository and then `pre-commit run all`